.bumpversion.cfg

```diff
@@ -12,10 +12,8 @@ search = :Version: {current_version}
 replace = :Version: {new_version}
 
 [bumpversion:file:package.json]
-search = "version": "{current_version}",
-replace = "version": "{new_version}",
-
-[bumpversion:file:src/objects/conf/api.py]
```

Comment:
@annashamray I think this was incorrectly part of bumpversion.cfg, it includes the API version which I think are separate from the version of the software, right?

Reply:
It's not an easy question actually. Maybe we need to discuss it with @joeribekker 
When we supported v1 and v2 of API at the same time we had Objects software version == Objects API v2 version

Now we dropped v1 of the API and released v3 of software and they started to deviate from each other. 
I personally don't like it. Objects API is mostly an Restful API with a little bit of admin interface, so it doesn't make a lot of sense to have separate versions. Moreover unlike OZ we don't have any standard to adhere to. 
I think having one version would be less confusing and more convenient for both developers and clients

But it's up to discussion

Reply:
Let's discuss it when Joeri is back indeed